server/service/src/processors/transfer/shipment/README.md

```diff
@@ -21,3 +23,24 @@ When an outbound shipment is updated and an inbound shipment is already generate
 ## Same site transfer (both stores on same site)
 
 You may want to refer to [requisition transfer docs](../requisition/README.md#same-site-transfer-both-stores-on-same-site) for example of how one instance of triggered processor can itself upsert records and process them in the next iteration
+
+## Returns
+
+Returns are invoices, with outbounds and inbounds, just the same as shipments. As such, they are transferred by the same processor.
+
+### Related documents
+
+Where shipments have a related Requisition (referring to the Requisition that the Outbound Shipment was generated from, or the Internal Order that an Inbound Shipment was received for), returns have an original Shipment.
+
+This is either:
+
+- the Inbound Shipment that an Outbound Return was created from
+- the Outbound Shipment that an Inbound Return is related to
+
+When an Inbound Return is generated from Outbound Return, we check for an Inbound Shipment linked (via the `original_shipment_id`) to that Outbound Return. When then check which Outbound Shipment is linked to that Inbound Shipment, and link the Inbound Return with its correct original Outbound Shipment.
+
+Manually created returns don't have an original Shipment.
+
+In the UI, these are shown in the `Related Documents` section on the Return detail page.
+
+Note that mSupply Desktop Outbound Return -> Open mSupply Inbound Return won't have an original shipment. `original_shipment_id` is new in Open mSupply, and isn't populated by mSupply Desktop. Thus, when in Open mSupply, viewing any Inbound Returns received from a customer who is using mSupply Desktop, there won't be any related documents.
```

Comment:
```suggestion
Note that mSupply Desktop Outbound Return -> Open mSupply Inbound Return won't have an original shipment. `original_shipment_id` is new in Open mSupply, and isn't populated by mSupply Desktop. Thus, there won't be any links in the `Related Document` section in Open mSupply for the Inbound Returns in Open mSupply.
```

Reply:
Just re-read this and was gonna ask you to remove the 2nd Open mSupply but you've already done it <3